steps/github/get-releases/main.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"context"
+
+	"github.com/shurcooL/githubv4"
+	"github.com/stackpulse/steps-sdk-go/env"
+	"github.com/stackpulse/steps-sdk-go/step"
+	"github.com/stackpulse/steps-sdk-go/step/codes"
+	"github.com/stackpulse/steps-sdk-go/step/status"
+
+	"golang.org/x/oauth2"
+)
+
+type Args struct {
+	Owner      string `env:"OWNER,required" envDefault:""`
+	Token      string `env:"TOKEN,required" envDefault:""`
+	Repository string `env:"REPOSITORY,required" envDefault:""`
+	NumOfTags  int    `env:"NUM_OF_LAST_TAGS" envDefault:"5"`
+}
+
+type GHCommand struct {
+	args Args
+}
+
+type Tag struct {
+	Name    string
+	Message string
+}
+
+type output struct {
+	Tags []Tag
+}
+
+func (output *output) AddItem(tag Tag) []Tag {
+	output.Tags = append(output.Tags, tag)
+	return output.Tags
+}
+
+func (gh *GHCommand) Init() status.Status {
+	err := env.Parse(&gh.args)
+	if err != nil {
+		return status.New(codes.InvalidArgument, err.Error(), "")
+	}
+	return status.OK
+}
+
+func (gh *GHCommand) Run() (step.Output, status.Status) {
+	var releaseQuery struct {
+		Repository struct {
+			Description string
+			Releases    struct {
+				Nodes []struct {
+					Name        string
+					CreatedAt   string
+					Description string
+				}
+			} `graphql:"releases(orderBy: {direction: DESC, field: CREATED_AT}, first: $num_of_tags)"`
+		} `graphql:"repository(owner: $owner, name: $name)"`
+	}
+
+	variables := map[string]interface{}{
+		"owner":       githubv4.String(gh.args.Owner),
+		"name":        githubv4.String(gh.args.Repository),
+		"num_of_tags": githubv4.Int(gh.args.NumOfTags),
+	}
+
+	src := oauth2.StaticTokenSource(
+		&oauth2.Token{AccessToken: gh.args.Token},
+	)
+	httpClient := oauth2.NewClient(context.Background(), src)
+
+	client := githubv4.NewClient(httpClient)
+	err := client.Query(context.Background(), &releaseQuery, variables)
+	if err != nil {
+		return nil, status.FromError(codes.Internal, "get-releases failed with error", err)
+	}
+	tags := []Tag{}
+	result := output{tags}
+	for _, edge := range releaseQuery.Repository.Releases.Nodes {
+		tag := Tag{edge.Name, edge.Description}
+		result.AddItem(tag)
+	}
+	return map[string]interface{}{"tags": result.Tags}, status.OK
```

Comment:
Can you check if it compiles? If yes, that's better
```suggestion
	return step.Output{"tags": result.Tags}, status.OK
```